### PR TITLE
Reword the authentication section in the generated README.md files

### PR DESCRIPTION
**Pull request by deyaaeldeen**
@joheredi please regenerate the readme files again :)

### CODE REVIEW

File: src/vanilla/Model/CodeModelTS.cs [hunk 1]
```diff
@@ -1065,14 +1065,13 @@ private void GenerateAvailableAPIVersions(MarkdownBuilder builder)
...
         {
-            builder.Section($"Authentication", () => 
-            {
-                builder.Line("One of the first steps in accessing the services using SDK is to authenticate the client. There are several methods to authenticate such as interactive login, etc. Refer [readme for @azure/ms-rest-nodeauth](https://www.npmjs.com/package/@azure/ms-rest-nodeauth) package for all options to authenticate the client.");
-                builder.Line("");
-                
-            });
             builder.Section($"nodejs - client creation and {GetSampleMethod()?.Name} {GetSampleMethodGroupName()} as an example written in TypeScript.", () =>
             {
+                builder.Section($"Authentication", () =>
```
Comment (ramya-rao-a) [inline src/vanilla/Model/CodeModelTS.cs:1070]: Do we need this "section"? Can we not add the line in the existing section above for `nodejs - client creation...`?

Comment (ramya-rao-a) [reply src/vanilla/Model/CodeModelTS.cs:1070]: On second thoughts we should add the new line under the "Install @azure/ms-rest-nodeauth"
 section

File: src/vanilla/Model/CodeModelTS.cs [hunk 2]
```diff
@@ -1065,14 +1065,13 @@ private void GenerateAvailableAPIVersions(MarkdownBuilder builder)
...
+                {
+                    builder.Line("There are several methods to authenticate the client, please refer to [the README.md file of @azure/ms-rest-nodeauth](https://www.npmjs.com/package/@azure/ms-rest-nodeauth) to learn more about supported authentication options.");
```
Comment (ramya-rao-a) [inline src/vanilla/Model/CodeModelTS.cs:1072]:
```suggestion
                    builder.Line("While the below sample uses the interactive login, other authentication options can be found in [the README.md file of @azure/ms-rest-nodeauth] package");
```

File: src/vanilla/Model/CodeModelTS.cs (hunk 1 above)
Comment (deyaaeldeen) [reply src/vanilla/Model/CodeModelTS.cs]: @ramya-rao-a please take another look.

File: src/vanilla/Model/CodeModelTS.cs [hunk 3]
```diff
@@ -1081,6 +1075,7 @@ private void GenerateHowToUseInNodeJs(MarkdownBuilder builder)
                 builder.Line();
                 builder.Section("Sample code", () =>
                 {
+                    builder.Line("While the below sample uses the interactive login, other authentication options can be found in [the README.md file of @azure/ms-rest-nodeauth](https://www.npmjs.com/package/@azure/ms-rest-nodeauth) package");
```
Comment (ramya-rao-a) [inline src/vanilla/Model/CodeModelTS.cs:1078]:
```suggestion
                    builder.Line("While the below sample uses the interactive login, other authentication options can be found in the [README.md file of @azure/ms-rest-nodeauth](https://www.npmjs.com/package/@azure/ms-rest-nodeauth) package");
```

**Review by ramya-rao-a — APPROVED** [review]
looks good

**Review by joheredi — APPROVED** [review]
Looks good!

File: test/azuremetadata/generated/Lro/README.md [hunk 4]
```diff
@@ -30,16 +26,17 @@ npm install @azure/ms-rest-nodeauth@"^3.0.0"
...
 
+While the below sample uses the interactive login, other authentication options can be found in the [README.md file of @azure/ms-rest-nodeauth](https://www.npmjs.com/package/@azure/ms-rest-nodeauth) package
 ```typescript
-import * as msRest from "@azure/ms-rest-js";
+const msRest = require("@azure/ms-rest-js")
 import * as msRestAzure from "@azure/ms-rest-azure-js";
-import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
-import { AutoRestLongRunningOperationTestService, AutoRestLongRunningOperationTestServiceModels, AutoRestLongRunningOperationTestServiceMappers } from "";
+const msRestNodeAuth = require("@azure/ms-rest-nodeauth")
+const { AutoRestLongRunningOperationTestService } = require("")
```
Comment (joheredi) [inline test/azuremetadata/generated/Lro/README.md:34]: This empty string is not caused by the changes in this PR and seems to be misconfiguration of the test generation script + test swagger.

**Pull request merged**